parser/parser.go

```diff
@@ -495,13 +496,9 @@ func rValue() (NodeExpr, error) {
 		return n, nil
 	}
 
-	if look_ahead != lexical.T_LITERAL && look_ahead != lexical.T_NUMERIC {
-		return nil, throwSyntaxError(lexical.T_LITERAL, look_ahead)
-	}
-
 	lexeme := lexical.CurrentLexeme
 	if look_ahead != lexical.T_LITERAL {
-		return nil, throwSyntaxError(lexical.T_AND, look_ahead)
+		return nil, errors.New("Only Literals and Date are allowed in `where` clause")
```

Comment:
When I run `select message from commits where 5 in message` I get this error but the message seems not make sense. I mean, 5 is a numeric literal and I'm using it in a where clause 😕 . What do you think about this case @jsixface ?

Reply:
Should we change it to sting literal?
I do not see any use case for using a number in where clause for any column in all the tables. I may be wrong, is there any?

Reply:
That's a good point. I gonna review that, but it make senses.